docs/proposals/26-workload-identity.md

```diff
@@ -405,9 +405,6 @@ A sample payload of a token will look like:
             "name": "<seed-name>",
             "uid": "<seed-uid>",
         },
-        "garden": {
-            "id": "<garden-cluster-identity>",
-        },
```

Comment:
Why is this not part of the tokens anymore?

Reply:
Because issuer (`iss` claim) already uniquely identify the garden cluster. In https://github.com/gardener/gardener/pull/10042#discussion_r1660611577, we decided to remove it for now. If needed, we can add it later.